proto/path/gateway.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package path;
+
+option go_package = "github.com/buildwithgrove/path/observation";
+
+// RequestType captures the origin of the request, which as of now is either:
+// A. Organic: a user sent the service request to a PATH instance, or
+// B. Synthetic: the Endpoint Hydrator generated the service request to get data on service endpoints.
+enum RequestType {
```

Comment:
I believe proto best practices indicate enums should be declared with an UNSPECIFIED value as the 0 tag. See here:
https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum

Also for consistency, WDYT about keeping enum naming in the format `REQUEST_TYPE_ORGANIC`, etc.?